app/components/atomic/galoy-primary-button/galoy-primary-button.tsx

```diff
@@ -26,8 +26,8 @@ export const GaloyPrimaryButton: FunctionComponent<PropsWithChildren<ButtonProps
 
 const useStyles = makeStyles((theme) => ({
   titleStyle: {
-    fontSize: 28,
-    lineHeight: 32,
+    fontSize: 27,
+    lineHeight: 28,
```

Comment:
line height should not be just one more pixel than fontSize I think. multine text would touch top to bottom otherwise afaiu. 

does this needs to be changed?